resources/js/store/modules/teams/actions.js

```diff
@@ -0,0 +1,154 @@
+import Vue from 'vue'
+import i18n from '../../../i18n'
+import router from "../../../routes";
+
+export const actions = {
+
+    /**
+     * The user wants to create a new team
+     */
+    async CREATE_NEW_TEAM (context, payload)
+    {
+        const title = i18n.t('notifications.success');
+        const body  = i18n.t('teams.created');
+
+        const error_title = i18n.t('notifications.error');
+        const error_body  = i18n.t('teams.create.max-created');
+
+        await axios.post('/teams/create', {
+            name: payload.name,
+            identifier: payload.identifier,
+            teamType: payload.teamType
+        })
+        .then(response => {
+            console.log('create_new_team', response);
+
+            if (response.data.success)
+            {
+                // show notification
+                Vue.$vToastify.success({
+                    title,
+                    body,
+                    position: 'top-right'
+                });
+
+                // add team to teams array
+                // context.commit('usersTeams', response.data.team);
+
+                // If user does not have an active team, assign
+                if (! context.rootState.user.user.team_id) {
+                    context.commit('userJoinTeam', response.data.team);
+                }
+            }
+
+            else
+            {
+                Vue.$vToastify.error({
+                    title: error_title,
+                    body: error_body,
+                    position: 'top-right'
+                });
+            }
+        })
+        .catch(error => {
+            console.error('create_new_team', error.response.data.errors);
+
+            context.commit('teamErrors', error.response.data.errors);
+        });
+    },
+
+    /**
+     * Get the combined effort for all of the users teams for this period
+     */
+    async GET_COMBINED_TEAM_EFFORT (context, payload)
+    {
+        await axios.get('/teams/combined-effort', {
```

Comment:
Should define constant for url.

Reply:
Why add an extra line?

Reply:
If we use this url in another place, every time we change it, we have to change it in 2 places.

Reply:
Makes sense. I don't plan on using in more than 1 location but if we do, we will refactor